### DISCUSSION TITLE
.Net, do we really need to use a preview version of .Net Aspire

### DISCUSSION THREAD

**Discussion by Expecho** [microsoft/autogen#3967] [General]
@rysweet question, AutoGen .Net has added .Net Aspire into the mix. I can see the added value in that. But by requiring an rc version

```
<PackageVersion Include="Aspire.Hosting" Version="9.0.0-rc.1.24511.1" />
```

things are getting difficult because we now need a preview version of Visual Studio to build the sources. Do we really need that or can we just for now use the stable version of Aspire?

> Visual Studio 2022 17.12 or higher supports .NET Aspire 9.

([source](https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9-release-candidate-1?tabs=unix&pivots=visual-studio))

**Comment by rysweet** [comment]
Good question. I was using the "WaitFor()" which is only in the new version. If you remove that you should be able to downgrade. Could substitute a health check. 

My thinking was that all the v9 stuff will move out of preview in two weeks, before we are finished with 0.4 of autogen. But since we don't yet make it easy to build with packages I see how this is a pain.

**Reply by Expecho** [reply]
Ok, thanks for your reaction. I'll pobably wait some more time or indeed downgrade.

**Reply by rysweet** [reply]
@LittleLittleCloud do you have any ideas on how to make this easier?
